sqliter-driver/src/linuxX64Main/kotlin/co/touchlab/sqliter/DatabaseFileContext.kt

```diff
@@ -0,0 +1,61 @@
+package co.touchlab.sqliter
+
+import co.touchlab.sqliter.internal.File
+import co.touchlab.sqliter.internal.FileFilter
+import kotlinx.cinterop.memScoped
+import kotlinx.cinterop.refTo
+import kotlinx.cinterop.toKString
+import platform.posix.NULL
+import platform.posix.PATH_MAX
+import platform.posix.getcwd
+import platform.posix.getenv
+
+actual object DatabaseFileContext {
+    actual fun deleteDatabase(name: String, basePath: String?) {
+        deleteDatabaseFile(databaseFile(name, basePath))
+    }
+
+    actual fun databasePath(databaseName: String, datapathPath: String?): String {
+        return databaseFile(databaseName, datapathPath).path
+    }
+
+    internal fun databaseDirPath(): String {
+        return getHomeDirPath() ?: memScoped {
```

Comment:
Actually here I'm not sure where the fallback path should lead to?
I just defined the `home` dir but of course this is suboptimal. But another arbitrary fallback dir is also IMO. But you did something similar on windows as well and one should configure the path anyways in most cases.
Maybe you have a preference where the fallback path should be? Or is `home` fine for you?